### PR TITLE
[issue21] add modal for adding books

### LINKED ISSUE #21: Create add book modal
**Is your feature request related to a problem? Please describe.**
N/A

**Describe how the solution should behave**
This modal should allow users to add new books. Validate the data using the third library like formik.

**Screenshots of the design/if any**
image

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by RawandDev**
## What?
Created the add book modal.
## Why?
So that users can add books.
## How to use?
You can open the modal and add a book.
## Screenshots 
A clear screenshots of your work.
## Gif
https://gyazo.com/cf6ac08b8115b643340a331951e77ec7

### CODE REVIEW

**Comment by RawandDev** [comment]
@MuhannedNoman can you please review this.

**Comment by MuhannedNoman** [comment]
Where is this exactly? How can I find what is done in this PR.

**Comment by RawandDev** [comment]
> Where is this exactly? How can I find what is done in this PR.

I included a GIF. You can click on the plus icon to show the modal

File: src/components/BookCard.jsx [hunk 1]
```diff
@@ -5,52 +5,77 @@ import { Link } from "react-router-dom";
...
+  //           key={index}
+  //           className="text-xs absolute top-9 left-16 bg-cards rounded-2xl px-2.5 py-0.5 transform transition ease-in duration-100 hover:-translate-y-0.5 opacity-80"
+  //         >
+  //           {genre}
+  //         </button>
+  //       );
+  //     default:
+  //       return;
+  //   }
+  // }
```
Comment (MuhannedNoman) [inline src/components/BookCard.jsx]: Remove unused code.

File: src/components/BookCard.jsx [hunk 2]
```diff
@@ -5,52 +5,77 @@ import { Link } from "react-router-dom";
...
 
   const genreButtons = genres.map((genre, index) => {
-    return <div key={index}>{createGenreButton(genre, index)}</div>;
+    // return createGenreButton(genre, index);
```
Comment (MuhannedNoman) [inline src/components/BookCard.jsx]: Remove.

**Review by MuhannedNoman — APPROVED** [review]
LGTM, implement suggestion,s and feel free to merge.

**Comment by rebaz36** [comment]
@RawandDev why are you not fixing these conflicts!!!!!!!!!!!!!!!!

**Comment by RawandDev** [comment]
> @RawandDev why are you not fixing these conflicts!!!!!!!!!!!!!!!!

Facing issues with pushing 😂

**Pull request merged**